scripting/jse_scoreboard.sp

```diff
@@ -12,6 +12,7 @@
 bool g_bJSECoreLoaded;
 
 int g_iScore[MAXPLAYERS + 1];
+int g_iLastTeam[MAXPLAYERS + 1];
```

Comment:
Possibly unneeded if we can get `oldteam` from the `player_team` event.

Reply:
That would return spectator when returning to a non spec team. The point of this variable is to know if they switched to the opposite team or not.